Zend/tests/ErrorException_getSeverity.phpt

```diff
@@ -0,0 +1,43 @@
+--TEST--
+final public int ErrorException::getSeverity ( void );
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br> - #phparty7 - @phpsp - novatec/2015 - sao paulo - br
+--FILE--
+<?php
+function exception_error_handler($severity) {
+ 
+    $message = "error exception, severity =>";
+    $code = 500;
+    $severity = 75;
```

Comment:
Like @krakjoe mentioned, please do not use magic numbers, severity here should mimic the E_* constants, so you could for example assign $severity to E_WARNING and just give the $code a value of 0 since you are testing for getSeverity here anyway